### PR TITLE
[NL Next] Tests for page config generation from Utterance

### PR DESCRIPTION
**Pull request by pradh**

### CODE REVIEW

File: server/lib/nl/nl_page_config_next.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
   SubjectPageConfig, Tile
 from lib.nl.nl_utterance import Utterance, ChartType, ChartSpec
 from lib.nl.nl_detection import Place, RankingType
-from lib.nl.nl_utils import get_sv_name
+from lib.nl import nl_utils
```
Comment (shifucun) [inline server/lib/nl/nl_page_config_next.py:21]: Shall we remove all the "nl_" file prefix under nl folder?

File: server/tests/lib/nl/nl_page_config_next_test.py [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+from parameterized import parameterized
+from typing import Dict
+from unittest.mock import patch
+
+from lib.nl import nl_utils, nl_utterance, nl_page_config_next
+from config.subject_page_pb2 import SubjectPageConfig
+from tests.lib.nl.test_utterance import PLACE_ONLY_UTTR, SIMPLE_UTTR, SIMPLE_WITH_SV_EXT_UTTR, \
+  SIMPLE_WITH_TOPIC_UTTR, COMPARISON_UTTR, CONTAINED_IN_UTTR, CORRELATION_UTTR, RANKING_UTTR
+
+PLACE_ONLY_CONFIG = """
```
Comment (shifucun) [inline server/tests/lib/nl/nl_page_config_next_test.py:27]: not necessary for this PR, is it better to have individual textproto test file for each of these?

**Review by shifucun — APPROVED** [review]
thanks for adding tests!

File: server/lib/nl/nl_page_config_next.py (hunk 1 above)
Comment (pradh) [reply server/lib/nl/nl_page_config_next.py:21]: Let me do in follow-up PR?  I want to rename nl_data_spec to fulfillment too.

File: server/tests/lib/nl/nl_page_config_next_test.py (hunk 2 above)
Comment (pradh) [reply server/tests/lib/nl/nl_page_config_next_test.py:27]: that would be nice, yes

**Pull request merged**